gui/chrome_tabs.py

```diff
@@ -227,7 +227,7 @@ def AddPage(self, win=None, title="Empty Tab", image: wx.Image=None, closeable=T
         self.page_container.Layout()
 
         self._pages.append(win)
-        self.tabs_container.AddTab(title, image, closeable)
```

Comment:
I don't think we need to translate variables like this. I feel like we should pre-translate anything thats passed into `AddPage()` and simply display it here. I'm not sure if it's possible to even do this since gettext won't know what this variable would be. Thoughts?